articles/quickstart/native/react-native/_includes/_config.md

```diff
@@ -0,0 +1,21 @@
+## Configure Callback URLs
+
+Callback URLs are the URLs that Auth0 invokes after the authentication process. Auth0 routes your application back to this URL and appends additional parameters to it, including a token. Since callback URLs can be manipulated, you will need to add your application's URL to your client's **Allowed Callback URLs** for security. This will enable Auth0 to recognize these URLs as valid. If omitted, authentication will not be successful.
+
+Go to your [Client's Dashboard](${manage_url}/#/applications/${account.clientId}/settings) and make sure that "Allowed Callback URLs" contains the following:
+
+#### iOS
+
+```text
```

Comment:
This isn't strictly true, as often the value contained in the `Info.plist` will reference another variable so can't be copied directly.  However I'll add a little something here.